### PR TITLE
Fix crash with a bunch of mods that add TrackedDataHandlers

### LINKED ISSUE #239: Registration of unknown TrackedDataHandlers causes crash. Does not occur with FAPI.
*Note, this is with the QFAPI bridge method. If needed, this issue may be moved to the QFAPI repository.*
*This is forwarded from <https://discord.com/channels/817576132726620200/1054993076873080892> on Quilt Community.*

## Steps to Reproduce (from thread)
 - Install Quilt, QSL and Terraform mods
 - Launch the game.
 - See crash originating from the registry claiming the unknown handler was already registered.

## Mod Pack as provided from the thread.
<https://cdn.discordapp.com/attachments/1054993076873080892/1055019484483686441/1.19.3_-_QSL_crash.zip>

## Logs
All logs were obtained on Quilt Loader to avoid any potential variance beyond what's necessary.

https://mclo.gs/RBQAmVb - Quilt Loader + Fabric API
https://mclo.gs/UYTpWIC - Quilt Loader + QSL

https://mclo.gs/Wt77CDE - Quilt Loader + QSL + `-Dquilt.debug.unknown_tracked_data_handler=true`
Lines of interest: https://mclo.gs/Wt77CDE#L469

### LINKED ISSUE #252: Add a new identifier for `LONG`
https://github.com/QuiltMC/quilt-standard-libraries/blob/772ef16add2c32b92c12890c5089e0b6f88ebcd2/library/entity/entity_networking/src/main/java/org/quiltmc/qsl/entity/networking/mixin/TrackedDataHandlerRegistryMixin.java#L74

a new tracked data handler for longs was added in 22w42a, but not this api. Should be fixed

### LINKED ISSUE #254: Traverse, Terrestria, Cinderscape causes crash on startup
Related issue: https://github.com/TerraformersMC/Traverse/issues/95

When I include above three mods, the game crashes on startup, and displays java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'terraform-wood-api-v1'! 

Traverse devs asked me to report this bug on Quilt team so I posted here

The crash report is attached below
https://mclo.gs/aWHKF0B

### PR DESCRIPTION
**Pull request by OroArmor**
Fixes #239, #252, and QuiltMC/quilt-standard-libraries#254

### CODE REVIEW

**Review by EnnuiL — APPROVED** [review]
Looks good! Also, yeah, now I know what to do now if the QSL testmod crashes with QFAPI :P

**Review by LambdAurora — APPROVED** [review]
Looks fine.

Fixing the field resetting would also be possible by doing assignation at head of clinit I think?
Unless not giving a value defaults to a `PUTSTATIC` still.

File: library/entity/entity_networking/src/main/java/org/quiltmc/qsl/entity/networking/mixin/TrackedDataHandlerRegistryMixin.java [hunk 1]
```diff
@@ -41,8 +41,14 @@ public class TrackedDataHandlerRegistryMixin {
...
 	@Unique
 	private static final boolean quilt$PRINT_WARNING = TriState.fromProperty("quilt.debug.unknown_tracked_data_handler").toBooleanOrElse(QuiltLoader.isDevelopmentEnvironment());
+
+	// WARNING: These fields aren't set until just before RETURN in <clinit>. Until then they are 0.
+	// I (OroArmor) am abusing that fact for the pastStaticInit field. Do what you wish to me, this is a good way of solving the problem.
+	// Why the two above fields work, my guess is that they are final.
 	@Unique
 	private static int quilt$currentUnknownId = 0;
+	@Unique
+	private static boolean pastStaticInit = true;
```
Comment (Leo40Git) [inline library/entity/entity_networking/src/main/java/org/quiltmc/qsl/entity/networking/mixin/TrackedDataHandlerRegistryMixin.java:51]:
```suggestion
	private static boolean quilt$pastStaticInit = true;
```

**Review by Leo40Git — APPROVED** [review]
LGTM, excluding one nitpick

**Comment by OroArmor** [comment]
> Looks fine.
> 
> Fixing the field resetting would also be possible by doing assignation at head of clinit I think?
> Unless not giving a value defaults to a `PUTSTATIC` still.

The problem is mixin puts it before return. So I can't control it. Maybe without `@Unique`

**Pull request merged**